integration/cli/build.configequality.test.ts

```diff
@@ -0,0 +1,424 @@
+          webhookTimeout: 11,
+          onError: "reject",
+          alwaysIgnore: {
+            namespaces: ["garbage"],
+          },
+          logLevel: "warning",
+          env: {
+            TEST: "env",
+          },
+          customLabels: {
+            namespace: {
+              test: "test",
+              value: "value",
+            },
+          },
+          rbacMode: "scoped",
+          rbac: [
+            {
+              apiGroups: ["test"],
+              resources: ["tests"],
+              verbs: ["get", "list"],
+            },
+          ],
+        };
+        await fs.writeFile(packageJson, JSON.stringify(config, null, 2));
+
+        // translate package.json values into ModuleConfig
+        peprUuid = `pepr-${config.pepr.uuid}`;
+        moduleConfig = config.pepr;
+        moduleConfig.appVersion = config.version;
+        moduleConfig.description = config.description;
+
+        // build module
+        const build = await pepr.cli(testModule, { cmd: `pepr build` });
+        expect(build.exitcode).toBe(0);
+        expect(build.stderr.join("").trim()).toBe("");
+        expect(build.stdout.join("").trim()).toContain("K8s resource for the module saved");
+
+        // load helm manifests
+        const chartDir = `${testModule}/dist/${moduleConfig.uuid}-chart`;
+        const helm = await pepr.cli(chartDir, { cmd: `helm template .` });
+        expect(build.exitcode).toBe(0);
+        expect(build.stderr.join("").trim()).toBe("");
+        expect(build.stdout.join("").trim()).not.toBe("");
+
+        const helmManifest = `${chartDir}/reified.yaml`;
+        await fs.writeFile(helmManifest, helm.stdout.join("\n"));
+        helmResources = await resource.fromFile(helmManifest);
+
+        // load pepr manifests
+        const peprManifest = `${testModule}/dist/pepr-module-${moduleConfig.uuid}.yaml`;
+        peprResources = await resource.fromFile(peprManifest);
+      }, time.toMs("2m"));
+
+      it("peprVersion", async () => {
+        const peprStringified = JSON.stringify(peprResources);
+        const helmStringified = JSON.stringify(helmResources);
+
+        expect(peprStringified.includes(moduleConfig.peprVersion)).toBe(false);
+        expect(helmStringified.includes(moduleConfig.peprVersion)).toBe(false);
+      });
+
+      it("appVersion", async () => {
+        const peprStringified = JSON.stringify(peprResources);
+        const helmStringified = JSON.stringify(helmResources);
+
+        expect(peprStringified.includes(moduleConfig.appVersion)).toBe(false);
+        expect(helmStringified.includes(moduleConfig.appVersion)).toBe(false);
+      });
+
+      // eslint-disable-next-line max-statements, complexity
+      it("uuid", async () => {
+        for (const clusterRole of [
+          resource.select(peprResources, kind.ClusterRole, peprUuid),
+          resource.select(helmResources, kind.ClusterRole, peprUuid),
+        ]) {
+          expect(clusterRole).toBeDefined();
+        }
+
```

Comment:
Tests after this line are unrelated to `uuid` but still part of the `uuid` test.

Reply:
...wha?  I don't think I follow.  All the assertions after that are for properties / values in the generated files that _use_ the config'd UUID (and there are a _lot_ of them).

Reply:
Ahh gotcha, the purpose of the test was lost among the 140-line implementation & same-y looking for loops to check properties. I wonder if there's an approach to this test along the lines of a parameterized test. Will spend a few cycles on the idea so we comply with linter rules.